setup.cfg

```diff
@@ -1,93 +1,8 @@
-# setuptools config
-# see http://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files
-[metadata]
-name = {{ project_name }}
-version = 0.1.0
-description = TODO
-long_description = file: README.rst
-url = https://github.com/maykinmedia/{{ project_name|lower }}
-project_urls =
-    Documentation = http://{{ project_name|lower }}.readthedocs.io/en/latest/
-    Changelog = https://github.com/maykinmedia/{{ project_name|lower }}/blob/main/CHANGELOG.rst
-    Bug Tracker = https://github.com/maykinmedia/{{ project_name|lower }}/issues
-    Source Code = https://github.com/maykinmedia/{{ project_name|lower }}
-license = MIT
-author = Maykin Media
-author_email = support@maykinmedia.nl
-keywords = TODO
-classifiers =
-    Development Status :: 3 - Alpha
-    Framework :: Django
-    Framework :: Django :: 3.2
-    Intended Audience :: Developers
-    Operating System :: Unix
-    Operating System :: MacOS
-    Operating System :: Microsoft :: Windows
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
-    Programming Language :: Python :: 3.10
-    Topic :: Software Development :: Libraries :: Python Modules
-
-[options]
-zip_safe = False
-include_package_data = True
-packages = find:
-install_requires =
-    django >= 3.2
-tests_require =
-    pytest
-    pytest-django
-    tox
-    isort
-    black
-    flake8
-
-[options.packages.find]
-include =
-    {{ project_name|lower }}
-    {{ project_name|lower }}.*
-
-[options.extras_require]
-tests =
-    pytest
-    pytest-django
-    tox
-    isort
-    black
-    flake8
-pep8 = flake8
-coverage = pytest-cov
-docs =
-    sphinx
-    sphinx-rtd-theme
-release =
-    bumpversion
-    twine
-
 # 3rd party configuration
 
 [aliases]
 test=pytest
```

Comment:
What should we do with this?

Reply:
can be dropped, I think everyone just calls `pytest` as they should, it was a compatibility layer for the old `python setup.py test`